apps/ledger-live-mobile/src/components/BottomModal.tsx

```diff
@@ -32,7 +32,7 @@ const BottomModal = ({
   const [open, setIsOpen] = useState(false);
   const modalLock = useSelector(isModalLockedSelector);
 
-  // workarround to make sure no double modal can be opened at same time
+  // workaround to make sure no double modal can be opened at same time
   useEffect(
     () => () => {
       isModalOpenedref = false;
```

Comment:
I feel like this is an issue when we try to open a second/third modal:
- the first modal opens, setting `isModalOpenedref` to `true`
- by closing the second modal we set `isModalOpenedref` at `false`
- so now if we try to open a third modal it will not close automatically because `isModalOpenedref === false`

Reply:
Well, maybe we should use a counter, and not a boolean ?
Each time we open a modal: +1, each time we close a modal: -1 ?